packages/shared-components/src/components/HeaderBar/HeaderBar.js

```diff
@@ -1,8 +1,11 @@
 import styled from 'styled-components';
+import irisTheme from 'theme/irisTheme';
 
 const HeaderBar = styled.div.withConfig({ displayName: 'HeaderBar' })`
   height: 50px;
-  border-bottom: 1px solid #c2c2c2;
+  display: flex;
+  align-items: center;
+  border-bottom: ${irisTheme.thicknesses.normal} solid ${irisTheme.colors.gray.border};
```

Comment:
Use `themeGet()`; it will adapt to the current theme rather than assuming the current theme is IRIS (a relatively safe assumption for now, but not future proof).

Reply:
ThemeGet is marked as deprecated. Ok, will switch to it

Reply:
Yeah. For some history, we deprecated themeGet because we wanted to switch to using css vars. However, we are still determining how we are supporting IE, so we should continue to use it for the meantime. The alternative would be to use css vars in styled components directly.